vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -223,10 +222,10 @@ func run(healthCheck *metrics.HealthCheck) {
 			resourceMetrics[apiv1.ResourceMemory] = *externalMemoryMetric
 		}
 		externalClientOptions := &input_metrics.ExternalClientOptions{ResourceMetrics: resourceMetrics, ContainerNameLabel: *ctrNameLabel}
-		klog.V(1).Infof("Using External Metrics: %+v", externalClientOptions)
+		klog.V(1).InfoS("Using External Metrics", "options", externalClientOptions)
 		source = input_metrics.NewExternalClient(config, clusterState, *externalClientOptions)
 	} else {
-		klog.V(1).Infof("Using Metrics Server.")
+		klog.V(1).InfoS("Using Metrics Server.")
```

Comment:
What's the style recommendation for InfoS?
Should logs still end with punctuation?

Reply:
Correct. they shouldn't https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#message-style-guidelines

Ill fix that